src/proxy/main.py

```diff
@@ -30,6 +30,15 @@
         detail="Please set the environment variable AZURE_STORAGE_CONNECTION_STRING",
     ) from key_error
 
+try:
+    sql_connection_string = os.environ["AZURE_SQL_CONNECTION_STRING"]
+except KeyError as key_error:
+    print("Please set the environment variable AZURE_SQL_CONNECTION_STRING")
+    raise HTTPException(
+        status_code=500,
+        detail="Please set the environment variable AZURE_SQL_CONNECTION_STRING",
+    ) from key_error
+
```

Comment:
We should only use s connection string locally, when deployed to Azure, we should use managed identity.